components/Footer/Footer.js

```diff
@@ -1,34 +1,33 @@
-import Image from "next/image";
+import Link from "next/link";
 import styled from "styled-components";
+import BookmarkIcon from "../Icons/Bookmark";
+import ScheduleIcon from "../Icons/Schedule";
 
-export default function Footer() {
+function Footer() {
   return (
     <StyledFooterList>
       <StyledIconContainer>
-        <StyledButton>
-          <Image
-            width={40}
-            height={40}
-            src={"/assets/icons8-calendar-24.png"}
-          />
-        </StyledButton>
-        <p>Schedule</p>
+        <Link href={"/"}>
+          <StyledButton>
+            <ScheduleIcon />
+            <p>Schedule</p>
+          </StyledButton>
+        </Link>
       </StyledIconContainer>
       <StyledIconContainer>
-        <StyledButton>
-          <Image
-            width={40}
-            height={40}
-            alt="bookmark"
-            src={"/assets/icons8-bookmark-outline.svg"}
-          />
-        </StyledButton>
-        <p>bookmark</p>
+        <Link href={"/"}>
```

Comment:
page 2 is not working. 
remember, wen are not using buttons (Links in this case) that dont work? 
I guess its oK to make the PR now, but the Userstory should contain the working routing.